lib/lynx/command/basic.rb

```diff
@@ -28,6 +28,7 @@ def option(option)
       end
 
       def authorize
+        @command << "--defaults-extra-file=#{config.credentials}" if config.credentials
         @command << "--user=#{config.username}" if config.username
         @command << "--password=#{config.password}" if config.password
```

Comment:
Should we remove `password` entirely?

Reply:
I didn't want to remove either user or password in case it was required somewhere else in the codebase or if overrides need to be specified (though I'd have to double check if --password works over a config file). I think there's something in the bin/setup script that also uses lynx to login to the local database with user "root" and an empty password, and so we would probably want to make a change in bin/setup before removing password support here. I can double check that though.

Reply:
Let's remove it and bump a major version. The gem isn't highly used so I would prefer to just make it impossible to pass the password this way if we can.

Reply:
did you want to keep user?